File: 4/04344.py
for i in range(int(input())):
    scores = list(map(int, input().split()))
    N = scores[0]
    scores = scores[1:]

    avg_score = sum(scores) / N
    cnt = 0
    for score in scores:
        if score > avg_score: cnt += 1

    print("%.3f" % (cnt / N * 100), end='')
    print('%')

File: 4/04344.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
	int C; cin >> C;

	for (int i = 0; i < C; i++) {
		int N; cin >> N;

		vector<int> scores;
		int sum_score = 0;
		for (int j = 0; j < N; j++) {
			int a; cin >> a;

			scores.push_back(a);
			sum_score += a;
		}

		double avg_score = (double)sum_score / (double)N;
		int cnt = 0;
		for (int j = 0; j < N; j++)
			if (scores[j] > avg_score) cnt++;

		double good_stud = (double)cnt / (double)N;
		printf("%.3f", good_stud * 100);
		cout << '%' << endl;
	}
}
